reconcile/dashdotdb_slo.py

```diff
@@ -82,12 +83,21 @@ def _post(self, service_slos: Iterable[ServiceSLO]) -> Response | None:
             if self.dry_run:
                 continue
 
+            if isnan(item.value):
+                LOG.info(
+                    f"{self.logmarker} Skipping SLO '{slo_name}' in SLO doc '{item.slo_doc_name}'"
+                    "as the obtained value is not a number"
+                )
```

Comment:
we can even capture that in the beginning to detect it in dry run https://github.com/app-sre/qontract-reconcile/blob/3a0cbc38491fa1a9fabb72524fbaf933b37e40a3/reconcile/dashdotdb_slo.py#L142-L146

Reply:
Ok, but i think we should still not fail

Reply:
+1

Reply:
Fixed in https://github.com/app-sre/qontract-reconcile/commit/dbc08b35e413f2de715929237f2f433bd2cfd884